tracking/ft/index.js

```diff
@@ -64,6 +64,9 @@ const oTrackingWrapper = {
 
 			const edition = document.querySelector('[data-next-edition]') ? document.querySelector('[data-next-edition]').getAttribute('data-next-edition') : null;
 			context.edition = edition;
+			if (/(?:^|;)\s*segmentID=([^;]+)/.test(document.cookie)) {
+				context.segmentID = RegExp['$1'];
```

Comment:
Ohhh, didn't know about this. Is it safe? (just going by the big 'Non-standard warning at the top of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/n)